docs/description.md

```diff
@@ -275,7 +275,6 @@ The available configuration options are:
 | timeout      | Timeout for REST and gRPC API requests.<br>Default: 5.0 seconds for REST and unlimited for gRPC |
 | host         | Host name of Qdrant service. If url and host are not set, defaults to 'localhost'.|
 | path         | Persistence path for QdrantLocal. Eg. `local_data/private_gpt/qdrant`|
-| force_disable_check_same_thread         | Force disable check_same_thread for QdrantLocal sqlite connection.|
```

Comment:
Why removing this?

Reply:
It was probably a mistake. This is my first ever contribution to a github repo :)